ext/mysql2/statement.c

```diff
@@ -225,12 +227,7 @@ static int my_big2ll(VALUE bignum, LONG_LONG *ptr)
          * only -0x8000000000000000 is safe if `len == 8 && nlz_bits == 0`
          */
         !rb_absint_singlebit_p(bignum)
-#elif defined(HAVE_RB_BIG_CMP)
```

Comment:
This case should still be needed for Ruby 1.9 and 2.0.

Reply:
Oh, I misread that `efli` as `ifdef`. Derp.

Reply:
Alright, this is fixed up and green now.